metrics/report/report_dockerfile/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # We would have used the 'verse' base, that already has some of the docs processing
 # installed, but I could not figure out how to add in the extra bits we needed to
 # the lite tex version is uses.
-FROM rocker/tidyverse
+FROM rocker/tidyverse:3.6.0
```

Comment:
The ideal would be for us to specify a version number in the [versions file](https://github.com/kata-containers/tests/blob/master/versions.yaml). However, that would require more rework as this `Dockerfile` would need to be generated from `Dockerfile.in`.

I'll leave the decision up to @grahamwhaley but I do think a comment in the file itself stating why this version is using would be useful too.

Reply:
Yeah, I think using the versions file for this would be overkill. We only use it in one place, and it's not part of the CI for instance.
A comment would be appropriate. afaict, what happened is `rocker` updated the base version of ubuntu, and that version does not have the `latex-xcolor` package any more - https://packages.ubuntu.com/xenial/latex-xcolor
I'm fine with pinning the rocker version - the comment should note we pin the base image to align with our package install requirements.